account_operating_unit_sequence/hooks.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2020 Ecosoft Co., Ltd (http://ecosoft.co.th/)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+from odoo import SUPERUSER_ID, api
+
+
+def assign_ou_sequences(cr, registry):
+    with api.Environment.manage():
+        env = api.Environment(cr, SUPERUSER_ID, {})
+        operating_unit_obj = env["operating.unit"]
+        sequence_obj = env["ir.sequence"]
+        for ou in operating_unit_obj.search([]):
+            payment_sequence = sequence_obj.create(
+                {
+                    "name": "Payment Transfer of {}".format(ou.name),
+                    "code": "account.payment.transfer.{}".format(ou.code),
+                    "prefix": "{}-TRANS".format(ou.code),
+                    "padding": 5,
+                    "company_id": ou.company_id.id,
+                },
+                {
+                    "name": "Payment Customer Invoice of {}".format(ou.name),
+                    "code": "account.payment.customer.invoice.{}".format(ou.code),
+                    "prefix": "{}-CUS.IN/".format(ou.code),
+                    "padding": 5,
+                    "company_id": ou.company_id.id,
+                },
+                {
+                    "name": "Payment Customer Refund of {}".format(ou.name),
+                    "code": "account.payment.customer.refund.{}".format(ou.code),
+                    "prefix": "{}-CUS.OUT/".format(ou.code),
+                    "padding": 5,
+                    "company_id": ou.company_id.id,
+                },
+                {
+                    "name": "Payment Suppier Invoice of {}".format(ou.name),
+                    "code": "account.payment.supplier.invoice.{}".format(ou.code),
+                    "prefix": "{}-SUPP.OUT/".format(ou.code),
+                    "padding": 5,
+                    "company_id": ou.company_id.id,
+                },
+                {
+                    "name": "Payment Suppier Refund of {}".format(ou.name),
+                    "code": "account.payment.supplier.refund.{}".format(ou.code),
+                    "prefix": "{}-SUPP.IN/".format(ou.code),
+                    "padding": 5,
+                    "company_id": ou.company_id.id,
+                },
```

Comment:
I install this module and raise error `TypeError: create() takes 2 positional arguments but 6 were given`. In creating multiple record you must to define list of dictionary such as [{'name': 'val1'}, {'name': 'val2'}, ...] https://subscription.packtpub.com/book/all_products/9781789618921/21/ch21lvl1sec218/creating-or-writing-multiple-records

![Selection_148](https://user-images.githubusercontent.com/24691983/78525299-f4c36280-7800-11ea-85e4-7383e03a7ec5.png)

Reply:
Edited.